src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -1,12 +1,31 @@
 {% trans_default_domain "locationview" %}
 {{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_location_actions', {'contentId': content.id, 'redirectLocationId': location.id})}) }}
 
+<dialog id="confirm-location-removal-{{ location.id }}" class="ez-modal">
+    <section>
+        <p>{{ 'confirm.remove.location'|trans()|desc('Are you sure you want to remove selected locations?') }}</p>
+    </section>
+    <footer>
+        <button type="button" class="ez-button ez-button-secondary ez-js-close-modal">
```

Comment:
You did it right @dpobel, but in that part the specification is not correct. And yes, I did not ask for that previously, but implicitly the UI Guidelines include this example as one of the three types of modals. 

Sorry for not asking you before about this, but I was on vacation at that time :-(